File: app/src/main/java/com/example/toothfairy/model/service/CuredInfoService.java
package com.example.toothfairy.model.service;

import com.example.toothfairy.entity.CuredInfo;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface CuredInfoService {
    @GET("api/cured/{age}")
    public Call<CuredInfo> getCuredInfo(@Path("age") int age);
}

File: app/src/main/java/com/example/toothfairy/model/service/CuredInfoService.kt
package com.example.toothfairy.model.service

import retrofit2.http.GET
import com.example.toothfairy.entity.CuredInfo
import retrofit2.Call
import retrofit2.http.Path

interface CuredInfoService {
    @GET("api/cured/{age}")
    fun getCuredInfo(@Path("age") age: Int): Call<CuredInfo?>?
}